### ISSUE TITLE
[Bug]: SSB performance decreased more than 25% of daily regression in standalone mode

### ISSUE THREAD

**Issue by aressu1985** [matrixorigin/matrixone#10236]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93): ca8f93c34
- Hardware parameters: 64C 256G
- OS type:
- Others:
```

### Actual Behavior

For daily regression on standlone mode on JUN 26th, the SSB cost decreased more than 25% percentage

for commit: 6c14bc30b (5.25)
[SSB 10G]
2023-06-25 00:11:16 The query q4.3  has be executed successfully,and cost: 169
2023-06-25 00:11:16 This test has been executed successfully
2023-06-25 00:11:16 The sum cost of turn[1] is 3487 ms.
2023-06-25 00:11:16 The sum cost of turn[2] is 2929 ms.
2023-06-25 00:11:16 The sum cost of turn[3] is 3040 ms.
2023-06-25 00:11:16 The sum cost of turn[4] is 2930 ms.
2023-06-25 00:11:16 The sum cost of turn[5] is 3004 ms.
**2023-06-25 00:11:16 The avg cost of all turns is 3078 ms.**

[SSB 100G]
2023-06-25 00:13:53 The query q4.3  has be executed successfully,and cost: 1493
2023-06-25 00:13:53 This test has been executed successfully
2023-06-25 00:13:53 The sum cost of turn[1] is 38589 ms.
2023-06-25 00:13:53 The sum cost of turn[2] is 32479 ms.
2023-06-25 00:13:53 The sum cost of turn[3] is 26050 ms.
2023-06-25 00:13:53 The sum cost of turn[4] is 31466 ms.
2023-06-25 00:13:53 The sum cost of turn[5] is 26339 ms.
**2023-06-25 00:13:53 The avg cost of all turns is 30984 ms.**

job link:
https://github.com/matrixorigin/mo-nightly-regression/actions/runs/5370639211/jobs/9742888840

but for commit: ca8f93c34(5.26)
[SSB 10G]
2023-06-26 01:33:24 The query q4.3  has be executed successfully,and cost: 484
2023-06-26 01:33:24 This test has been executed successfully
2023-06-26 01:33:25 The sum cost of turn[1] is 6736 ms.
2023-06-26 01:33:25 The sum cost of turn[2] is 6069 ms.
2023-06-26 01:33:25 The sum cost of turn[3] is 5816 ms.
2023-06-26 01:33:25 The sum cost of turn[4] is 5650 ms.
2023-06-26 01:33:25 The sum cost of turn[5] is 5558 ms.
**2023-06-26 01:33:25 The avg cost of all turns is 5965 ms.**

[SSB 100G]
2023-06-26 01:38:30 The query q4.3  has be executed successfully,and cost: 5282
2023-06-26 01:38:30 This test has been executed successfully
2023-06-26 01:38:30 The sum cost of turn[1] is 80322 ms.
2023-06-26 01:38:30 The sum cost of turn[2] is 57699 ms.
2023-06-26 01:38:30 The sum cost of turn[3] is 55507 ms.
2023-06-26 01:38:30 The sum cost of turn[4] is 53646 ms.
2023-06-26 01:38:30 The sum cost of turn[5] is 55869 ms.
**2023-06-26 01:38:30 The avg cost of all turns is 60608 ms.**

job link:
https://github.com/matrixorigin/mo-nightly-regression/actions/runs/5365116632/jobs/9733755241

### Steps to Reproduce

```Markdown
just run ssb test
```

**Comment by badboynt1** [comment]
caused by this pr: https://github.com/matrixorigin/matrixone/pull/10203

@nnsgmsone

**Comment by nnsgmsone** [comment]
fixed by #10247

**Comment by aressu1985** [comment]
fixed
